tests/unit/test_source_detection_step1.py

```
@@ -0,0 +1,84 @@
+    En el artículo de The Guardian se menciona que €2.5 millones...
+    Fuente: World Bank Report 2024, página 15.
+    """
+    
+    # Test without timestamps (pseudo-segments)
+    result = service.detect_sources_in_video(
+        video_url="https://youtube.com/watch?v=test123",
+        transcript=test_transcript,
+        with_timestamps=False
+    )
+    
+    print(f"✅ Resultado del análisis:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)